assets/xml/objects/object_owl.xml

```diff
@@ -1,95 +1,121 @@
-        <DList Name="object_owl_DL_00FE30" Offset="0xFE30" />
-        <DList Name="object_owl_DL_00FEF8" Offset="0xFEF8" />
-        <DList Name="object_owl_DL_00FFE8" Offset="0xFFE8" />
-        <DList Name="object_owl_DL_010180" Offset="0x10180" />
-        <DList Name="object_owl_DL_010248" Offset="0x10248" />
-        <DList Name="object_owl_DL_010338" Offset="0x10338" />
-        <Limb Name="object_owl_Standardlimb_0104D0" Type="Standard" EnumName="OBJECT_OWL_2_LIMB_01" Offset="0x104D0" />
-        <Limb Name="object_owl_Standardlimb_0104DC" Type="Standard" EnumName="OBJECT_OWL_2_LIMB_02" Offset="0x104DC" />
-        <Limb Name="object_owl_Standardlimb_0104E8" Type="Standard" EnumName="OBJECT_OWL_2_LIMB_03" Offset="0x104E8" />
-        <Limb Name="object_owl_Standardlimb_0104F4" Type="Standard" EnumName="OBJECT_OWL_2_LIMB_04" Offset="0x104F4" />
-        <Limb Name="object_owl_Standardlimb_010500" Type="Standard" EnumName="OBJECT_OWL_2_LIMB_05" Offset="0x10500" />
-        <Limb Name="object_owl_Standardlimb_01050C" Type="Standard" EnumName="OBJECT_OWL_2_LIMB_06" Offset="0x1050C" />
-        <Limb Name="object_owl_Standardlimb_010518" Type="Standard" EnumName="OBJECT_OWL_2_LIMB_07" Offset="0x10518" />
-        <Limb Name="object_owl_Standardlimb_010524" Type="Standard" EnumName="OBJECT_OWL_2_LIMB_08" Offset="0x10524" />
-        <Limb Name="object_owl_Standardlimb_010530" Type="Standard" EnumName="OBJECT_OWL_2_LIMB_09" Offset="0x10530" />
-        <Limb Name="object_owl_Standardlimb_01053C" Type="Standard" EnumName="OBJECT_OWL_2_LIMB_0A" Offset="0x1053C" />
-        <Limb Name="object_owl_Standardlimb_010548" Type="Standard" EnumName="OBJECT_OWL_2_LIMB_0B" Offset="0x10548" />
-        <Limb Name="object_owl_Standardlimb_010554" Type="Standard" EnumName="OBJECT_OWL_2_LIMB_0C" Offset="0x10554" />
-        <Limb Name="object_owl_Standardlimb_010560" Type="Standard" EnumName="OBJECT_OWL_2_LIMB_0D" Offset="0x10560" />
-        <Limb Name="object_owl_Standardlimb_01056C" Type="Standard" EnumName="OBJECT_OWL_2_LIMB_0E" Offset="0x1056C" />
-        <Limb Name="object_owl_Standardlimb_010578" Type="Standard" EnumName="OBJECT_OWL_2_LIMB_0F" Offset="0x10578" />
-        <Skeleton Name="object_owl_Skel_0105C0" Type="Flex" LimbType="Standard" LimbNone="OBJECT_OWL_2_LIMB_NONE" LimbMax="OBJECT_OWL_2_LIMB_MAX" EnumName="ObjectOwl2Limb" Offset="0x105C0" />
+        <!-- Animation -->
+        <Animation Name="gOwlTakeoffAnim" Offset="0x1168" />
+        
+        <!-- Single feather -->
+        <DList Name="gOwlFeatherDL" Offset="0x1200" />
+        <Texture Name="gOwlFeatherTex" OutName="owl_feather" Format="rgba16" Width="16" Height="32" Offset="0x1290" />
+        
+        <!-- Animation -->
+        <Animation Name="gOwlFlyAnim" Offset="0x1ADC" />
+        
+        <!-- Owl Flying DList -->
+        <DList Name="gOwlFlyingTorsoDL" Offset="0x5050" />
+        <DList Name="gOwlFlyingNeckDL" Offset="0x51C8" />
```

Comment:
This feels more like an "upper body" limb than a "neck" limb, just looking at the model:
![image](https://github.com/zeldaret/mm/assets/12245827/bd3e04d1-658b-47db-af69-0daf6e395be8)